modules/data.land/R/extract_soil_nc.R

```diff
@@ -113,10 +113,10 @@ extract_soil_nc <- function(in.file,outdir,lat,lon){
 soil.units <- function(varname = NA){
   variables <- as.data.frame(matrix(c("soil_depth","m",
                                       "soil_cec","meq/100g",
-                                      "volume_fraction_of_clay_in_soil","m3 m-3",
-                                      "volume_fraction_of_sand_in_soil","m3 m-3",
-                                      "volume_fraction_of_silt_in_soil","m3 m-3",
-                                      "volume_fraction_of_gravel_in_soil","m3 m-3",
+                                      "fraction_of_clay_in_soil","m3 m-3",
+                                      "fraction_of_sand_in_soil","m3 m-3",
+                                      "fraction_of_silt_in_soil","m3 m-3",
+                                      "fraction_of_gravel_in_soil","m3 m-3",
```

Comment:
So does the m3 m-3 unit allow for mass? It seems like it should be kg, or are we keeping it mass/volume-agnostic?

Reply:
In bety it's unitless, right?

Reply:
Good catch, I was focused on the variable names and completely forgot about the units. Yes bety is correct (unitless)